packages/admin/resources/views/components/page.blade.php

```diff
@@ -69,9 +69,11 @@
         >
             @if ($action)
                 @if ($action->isModalCentered())
-                    <x-slot name="heading">
-                        {{ $action->getModalHeading() }}
-                    </x-slot>
+                    @if ($heading = $action->getModalHeading())
+                        <x-slot name="heading">
+                            {{ $action->getModalHeading() }}
```

Comment:
These should all be updated to use the set variable:

```suggestion
                            {{ $heading }}
```